src/DynamoCore/Core/DynamoMigrator.cs

```diff
@@ -401,4 +424,36 @@ protected override DynamoMigratorBase MigrateFrom(DynamoMigratorBase sourceMigra
             return base.MigrateFrom(sourceMigrator);
         }
     }
+
+    class MigratorPathResolver : IPathResolver
+    {
+        private readonly IEnumerable<string> emptyList = new List<string>();
+        public MigratorPathResolver(string userDataRoot)
+        {
+            UserDataRootFolder = userDataRoot;
+        }
+
+        public IEnumerable<string> AdditionalResolutionPaths
+        {
+            get { return emptyList; }
+        }
+
+        public IEnumerable<string> AdditionalNodeDirectories
+        {
+            get { return emptyList; }
+        }
+
+        public IEnumerable<string> PreloadedLibraryPaths
+        {
+            get { return emptyList; }
+        }
+
+        public string UserDataRootFolder { get; set; }
```

Comment:
The class is internal, so doesn't make much difference. This class is not supposed to be used by any other client.